std/algorithm/comparison.d

```diff
@@ -560,7 +560,7 @@ if (isInputRange!R1 && isInputRange!R2 && !(isSomeString!R1 && isSomeString!R2))
     }
 }
 
-// Specialization for strings (for speed purposes)
+/// Ditto Specialization for strings (for speed purposes)
 int cmp(alias pred = "a < b", R1, R2)(R1 r1, R2 r2) if (isSomeString!R1 && isSomeString!R2)
```

Comment:
Does making this show up in documentation really add any documentation value? Also, `/// Ditto` *and* a unique description in the same comment? How does that work?